src/renderer/qss/js/qssServiceButtons.js

```diff
@@ -206,6 +206,29 @@
         openUSB.fire();
     };
 
+    /**
+     * Inherits from `gpii.qss.buttonPresenter` and handles interactions with the "launch .exe file"
+     * QSS button.
+     */
+    fluid.defaults("gpii.qss.launchExePresenter", {
```

Comment:
This default is named generically, but it is hard-wired to only launch the docuMorphExecutable option from the site config. Either find a way to express what it launches parametrically (e.g. by having the particular instance of it supplying an option "docuMorphExecutable" or else the resolved siteConfig option), or rename it

Reply:
The presenter is renamed properly, and now its using a universal function to execute the file.